### ISSUE TITLE
open browser window for more accessibility info doesn't work

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#44867]
This action is also hard coded atm. it should be an `Action`.

![Screenshot 2024-05-21 at 12 29 11 PM](https://github-com/microsoft/vscode/assets/29464607/64afdd75-b6bb-44f9-8097-61c10a673427)
